sigstore/_cli.py

```diff
@@ -1167,7 +1158,27 @@ def _verify_common(
         return None
 
 
-def _get_identity(args: argparse.Namespace) -> Optional[IdentityToken]:
+def _get_trust_config(args: argparse.Namespace) -> ClientTrustConfig:
+    """
+    Return the client trust configuration (Sigstore service URLs, key material and lifetimes)
+
+    The configuration may come from explicit argument (--trust-config) or from the TUF
+    repository of the used Sigstore instance.
+    """
+    # Some commands do not have offline argument, parse carefully
+    offline = args.offline if "offline" in args else False
```

Comment:
Nitpick: I think this is a little more idiomatic:

```suggestion
    # Not all commands provide --offline
    offline = getattr(args, "offline", False)
```

(We could also make `--offline` a global argument to avoid this entirely -- I would have no objection to that 🙂)